### PR TITLE
Subdao reorg

### PR DESCRIPTION
**Pull request by allenan**
Reorganizes explorer navigation around the concept of subdaos, making a clearer distinction between HNT (validators), IOT (hotspots) and MOBILE (5g hotspots). I expect more data and widgets to be added especially in the MOBILE section

### CODE REVIEW

File: components/InfoBox/Iot/StatisticsPane.js [hunk 1]
```diff
@@ -53,7 +66,7 @@ const StatisticsPane = () => {
         value={makers?.length}
         subtitle={`Latest: ${maxBy(makers, 'id')?.name}`}
         isLoading={!makers}
-        linkTo="/hotspots/makers"
+        linkTo="/iot/makers"
```
Comment (tyler-whitman) [inline components/InfoBox/Iot/StatisticsPane.js:69]: There is another usage of `/hotspots/makers` in the breadcrumbs for `AccountDetailsInfoBox`. Im wondering if that should now be `/iot/makers`? https://github.com/helium/explorer/blob/v2/components/InfoBox/AccountDetailsInfoBox.js#L59

File: components/InfoBox/OverviewInfoBox.js [hunk 2]
```diff
@@ -69,6 +115,23 @@ const OverviewInfoBox = () => {
           isLoading={!market}
           linkTo="/market"
         />
+        <Widget
+          title="Market Cap"
```
Comment (tyler-whitman) [inline components/InfoBox/OverviewInfoBox.js]: Should this be `HNT Market Cap`?

File: components/InfoBox/OverviewInfoBox.js [hunk 3]
```diff
@@ -48,17 +51,60 @@ const OverviewInfoBox = () => {
...
+              title="5G Radios"
+              series={mobileStats?.count}
+              isLoading={!mobileStats}
+              transparent
+            />
+          }
+          linkTo="/mobile"
         />
         <Widget
           title="Market Price"
```
Comment (tyler-whitman) [inline components/InfoBox/OverviewInfoBox.js]: Should this be `HNT Market Price`?

File: components/InfoBox/OverviewInfoBox.js [hunk 4]
```diff
@@ -1,38 +1,41 @@
 import { round } from 'lodash'
 import InfoBox from './InfoBox'
 import TrendWidget from '../Widgets/TrendWidget'
-import StatWidget from '../Widgets/StatWidget'
 import useApi from '../../hooks/useApi'
 import InfoBoxPaneContainer from './Common/InfoBoxPaneContainer'
 import { formatLargeNumber } from '../../utils/format'
 import Widget from '../Widgets/Widget'
+import DaoWidget from '../Widgets/DaoWidget'
```
Comment (tyler-whitman) [inline components/InfoBox/OverviewInfoBox.js:8]: One issue I saw when I was running ts components earlier was when I imported them I was getting an error if I didn't import with the extension `'../Widgets/DaoWidget.tsx'`. This seems to work though so maybe something was going on with my local build.

File: next.config.js [hunk 5]
```diff
@@ -11,15 +11,34 @@ module.exports = {
       },
     ]
   },
+  async redirects() {
+    return [
+      {
+        source: '/hotspots',
+        destination: '/iot',
+        permanent: false,
```
Comment (tyler-whitman) [inline next.config.js:19]: What would permanent do? So this will cause old links to still work properly right?

**Review by tyler-whitman — APPROVED** [review]
Looks good, do we need to wait until mobile is out to merge this?

File: next.config.js (hunk 5 above)
Comment (allenan) [reply next.config.js:19]: it issues a 302 vs 301 redirect. I just figured if we found something wrong with the redirects, we could fix it without everyone's browser having cached the redirect. If all is good after a couple days, we could switch them to permanent/301

File: components/InfoBox/OverviewInfoBox.js (hunk 3 above)
Comment (allenan) [reply components/InfoBox/OverviewInfoBox.js]: sure we can specify HNT on these

File: components/InfoBox/Iot/StatisticsPane.js (hunk 1 above)
Comment (allenan) [reply components/InfoBox/Iot/StatisticsPane.js:69]: yes probably let me look

File: components/InfoBox/OverviewInfoBox.js (hunk 4 above)
Comment (allenan) [reply components/InfoBox/OverviewInfoBox.js:8]: hm maybe something with your dependencies or a nodejs version difference?

**Pull request merged**